views/db/advisories/index.erb

```diff
@@ -12,3 +12,5 @@
 <% @advisories.each do |advisory| %>
   <p><a href="/db/advisories/<%=h advisory.id %>"><%=h advisory %></a></p>
 <% end %>
+
+<%= pagy_bulma_nav(@pagy) if @pagy.pages > 1 %>
```

Comment:
This tiny bit of logic should be extracted into a partial, that way if we want to add to or change the pagination, we can change it in one place.